src/main/java/de/bsi/secvisogram/csaf_cms_backend/rest/AdvisoryController.java

```diff
@@ -656,13 +686,26 @@ public EntityCreateResponse addAnswer(
             @Parameter(
                     in = ParameterIn.PATH,
                     description = "The ID of the comment to add the answer to."
-            ) long commentId,
-            @RequestBody String answerText
+            ) String commentId,
+            @RequestBody String newAnswerJson
```

Comment:
It would be better to use a dedicated Request Object for the newAnswerJson.
It is easier to understand, wich json object is used. And it is validated that only defined value are transfered to the server